cli/functionary/package.py

```diff
@@ -51,6 +55,13 @@ def create_cmd(ctx, language, name, output_directory):
 
     Create an example function in the specified language.
     """
+    if "/" in name:
+        ex_path = name.rsplit("/", 1)[0]
+        ex_name = name.rsplit("/", 1)[1]
+        raise click.ClickException(
+            "Your package name looks like a path. Try using this command instead:"
+            f"\n       functionary package create -l {language} -o {ex_path}/ {ex_name}"
```

Comment:
Let's drop the trailing slash in `{ex_path}/`.  With it there, if you copy and paste the command as it gets output, you'll end up with a "//" in the path that gets output at the end.